include/aspect/global.h

```diff
@@ -39,6 +39,7 @@
 
 #include <deal.II/base/mpi.h>
 #include <deal.II/base/multithread_info.h>
+#include <deal.II/base/std_cxx11/unique_ptr.h>
```

Comment:
it was unconditionally included in compat.h, but you are right, it seems to work without. Let's see if the tester agrees.